java/src/main/java/DemoTransit.java

```diff
@@ -384,7 +386,7 @@ public String CreateJWTSaveURL(String issuerId, String classSuffix, String userI
                             .setDefaultValue(
                                 new TranslatedString()
                                     .setLanguage("en-US")
-                                    .setValue("Fare name"))));
+                                    .setValue("Fare name")))));
```

Comment:
Did this compile correctly for you?

Reply:
No. Just checked. Thanks for the change 😀
Initially for Java I was setting the trip type at the [end of the object creation](https://github.com/google-pay/wallet-samples/commit/78570f4f4288e1e7fe18113c356986bc6f214766). After moving it I mistakenly left this brace.